src/components/network/line-layer.js

```diff
@@ -81,24 +81,37 @@ class LineLayer extends CompositeLayer {
 
     renderLayers() {
         const layers = [];
-
         // lines : create one layer per nominal voltage, starting from higher to lower nominal voltage
         this.state.compositeData.forEach(compositeData => {
             const color = this.props.getNominalVoltageColor(compositeData.nominalVoltage);
 
+            const doDash = line => {
+                return line.p1 == null || line.p2 == null;
+            };
+            const isDisconnected = line => {
+                return line.p1 == null && line.p2 == null
+            };
+            const noDashArray=[0,0];
```

Comment:
You should declare it outside of render method

Reply:
Javascript why u so tricky >_>